proto/context.proto

```diff
@@ -4,7 +4,20 @@ import "proto/user_id.proto";
 
```

Comment:
Easier for me to keep as-is since I'm still working on my Git-fu... Not sure if this addresses the root of your comment, but you can also click "Viewed" in GitHub to tick off individual files as LGTM